kasa/smartcam/modules/homekit.py

```diff
@@ -0,0 +1,24 @@
+"""Implementation of homekit module.
+
+Currently only useful to check if device supports homekit.
+"""
+
+from __future__ import annotations
+
+from ..smartcammodule import SmartCamModule
+
+
+class HomeKit(SmartCamModule):
+    """Implementation of homekit module.
+
+    Currently only useful to check if device supports homekit.
+    """
+
+    # Rename/Remove NAME when implementing common interface
+    NAME = "SmartHomeKit"
```

Comment:
If there is no downside for having a duplicate name (considering two types of modules won't be initialized), I'd skip this.

Reply:
Ok, went back to the regular names for this one as I can't quite recall what I was hoping to achieve w.r.t migration path for common modules.